omod/src/main/webapp/resources/scripts/patientsearch/patientSearchWidget.js

```diff
@@ -50,7 +50,7 @@ function PatientSearchWidget(configuration){
     });
     tableHtml += '</tr></thead><tbody></tbody></table>';
 
-    var spinnerImage = '<span><img class="search-spinner" src="'+emr.resourceLink('uicommons', 'images/spinner.gif')+'" /></span>';
+    var spinnerImage = '<span id="spinner-image"><img class="search-spinner" src="'+emr.resourceLink('uicommons', 'images/spinner.gif')+'" /></span>';
```

Comment:
Do we need to add this id?  Can't we just use a class-based selector with "search-spinner" ?